cdap-ui/app/cdap/services/datasource/index.js

```diff
@@ -115,9 +123,20 @@ export default class Datasource {
     });
     this.eventEmitter.on(WINDOW_ON_FOCUS, this.resumePoll.bind(this));
     this.eventEmitter.on(WINDOW_ON_BLUR, this.pausePoll.bind(this));
+    if (!this.excludeFromHealthCheck) {
+      SystemDelayStore.dispatch({
+        type: SystemDelayActions.registerDataSource,
+        payload: this,
+      });
+    }
+  }
+
+  getBindings(){
```

Comment:
this should probably have a better function signature. Bindings is a map, but the return value here is an array.

also nit: space after `()`.

Reply:
renamed the function and fixed the spacing issue